docs/src/3.1.Worldview.mdx

```diff
@@ -2,22 +2,23 @@
-| `onClick`             | `MouseHandler`                                  |                        |                                                                                                         |
+| Name                  | Type                                            | Default                | Description                                                                                                            |
+| --------------------- | ----------------------------------------------- | ---------------------- | ---------------------------------------------------------------------------------------------------------------------- |
+| `cameraState`         | `$Shape<CameraState>`                           |                        | control the [cameraState](#/docs/api/camera), must use with `onCameraStateChange`                                      |
+| `onCameraStateChange` | `(CameraState) => void`                         |                        | callback whenever the camera state changes                                                                             |
+| `defaultCameraState`  | `$Shape<CameraState>`                           | `DEFAULT_CAMERA_STATE` | pass in this prop to turn Worldview to an uncontrolled component. `onCameraStateChange` will be ignored                |
+| `keyMap`              | `CameraKeyMap`                                  | {}                     | override default Worldview hotkeys                                                                                     |
+| `backgroundColor`     | `Vec4`                                          | `[0,0,0,1]`            | change the scene background color                                                                                      |
+| `hitmapOnMouseMove`   | `boolean`                                       |                        | if enabled, the clicked objectId will be returned from the mouse event handler                                         |
+| `children`            | `React.Node`                                    |                        | regl components to render inside the Worldview                                                                         |
+| `style`               | `{ [styleAttribute: string]: number | string }` | {}                     | html style object for the Worldview root element                                                                       |
+| `showDebug`           | `boolean`                                       |                        | show debug information                                                                                                 |
+| `mouseClickRadius`    | `number`                                        | 3                      | if the mouse has moved distance beyond the mouseClickRadius limit, the click event handler isn't going to be triggered |
```

Comment:
might be nice to mention in the docs the number is number of pixels in screen space.  There's a very small potential one could think it means the distance in the scene.